backtester/config/examples/rsi-api-candles.strat

```diff
@@ -0,0 +1,96 @@
+{
+ "nickname": "TestGenerateRSICandleAPICustomSettingsStrat",
+ "strategy-settings": {
+  "name": "rsi",
+  "is-multi-currency": false,
+  "custom-settings": {
+   "rsi-high": 70,
+   "rsi-low": 30,
+   "rsi-period": 14
+  }
+ },
+ "currency-settings": [
+  {
+   "exchange-name": "binance",
+   "asset": "spot",
+   "base": "BTC",
+   "quote": "USDT",
+   "initial-funds": 1000000,
+   "leverage": {
+    "can-use-leverage": false,
+    "maximum-leverage-ratio": 0,
+    "maximum-leverage-rate": 0
+   },
+   "buy-side": {
+    "minimum-size": 0.1,
+    "maximum-size": 1,
+    "maximum-total": 10000
+   },
+   "sell-side": {
+    "minimum-size": 0.1,
+    "maximum-size": 1,
+    "maximum-total": 10000
+   },
+   "min-slippage-percent": 0,
+   "max-slippage-percent": 0,
+   "maker-fee-override": 0.002,
+   "taker-fee-override": 0.001,
+   "maximum-holdings-ratio": 0
+  },
+  {
+   "exchange-name": "binance",
+   "asset": "spot",
+   "base": "ETH",
+   "quote": "USDT",
+   "initial-funds": 100000,
+   "leverage": {
+    "can-use-leverage": false,
+    "maximum-leverage-ratio": 0,
+    "maximum-leverage-rate": 0
+   },
+   "buy-side": {
+    "minimum-size": 0.1,
+    "maximum-size": 1,
+    "maximum-total": 10000
+   },
+   "sell-side": {
+    "minimum-size": 0.1,
+    "maximum-size": 1,
+    "maximum-total": 10000
+   },
+   "min-slippage-percent": 0,
+   "max-slippage-percent": 0,
+   "maker-fee-override": 0.002,
+   "taker-fee-override": 0.001,
+   "maximum-holdings-ratio": 0
+  }
+ ],
+ "data-settings": {
+  "interval": 86400000000000,
```

Comment:
The field is just the json representation of `time.Duration` which we use in GCT config as well. I understand that it can be annoying to convert but I don't think I'll be creating my own data type for it unless its for all of GCT.

As discussed, I think the path I'll take will be to continue to flag the data as missing so no actions can be taken, but perhaps for graph representation and statistics calculation, set the price to the previous candle when data is missing.

Reply:
👍